docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
```

Comment:
Are you willing to merge this file? If so, I have few questions:
- I don't see any `./data/license.jwt`, so I guess we should leave some instructions, at least.
- Where's the `renderer` image name taken from? The `Makefile` refers to `grafana/grafana-image-renderer:dev`, so I guess we should either change it accordingly, or point to the Dockerfile from the `docker-compose.yaml`.

Reply:
My bad, used this for local testing and committed it accidentally, will remove.